### ISSUE TITLE
Conveyor Belt turns into something else when broken.

### ISSUE THREAD

**Issue by idnthvn** [BluSunrize/ImmersiveEngineering#3511]
Playing on the Industrial Pack and helping the owner while he is out. 
When placing conveyor belts, all works well. 
When you break them, sometimes they get frozen there and never brake (the belt rendering motion goes away and you get a black and purple texture where the belt rides)
I noticed that after i broke them, i actually had to use world edit to get rid of them.
Also noticed that if i was lucky enough to break them, that they do not break into a conveyor belt.
They have the following name now and do not work anymore as part of the structures 
"tile.immersiceenginerring.conveyor..name" i assume there is more to that, but does not show on the NEI system.
With crafttweaker command /crafttweaker hand
i get the following 

Good conveyor belt = <immersiceengineering:conveyor>.withTag({conveyorType:"immersiveengineering:conveyor"})

the bad conveyor belt = <immersiceengineering:conveyor>

So looks like when broken, the tag gets lost.
Hope this can be an easy fix for in the next release.
Thanks

**Comment by BluSunrize** [comment]
1. What are you breaking them with when this issue appears?

2. Can you reproduce this reliably in a test world?

3. Please provide a mod list for that pack.

**Comment by idnthvn** [comment]
Sorry for the long delay. I stepped away for a while. 
So in single player, i cannot replicate. 
The only difference between the server and that is the SpongeForge & Ti-Quality that is installed. 
The server was completely wipped clean and every mod was updated to the newest as of January 1st 2020. 
Problem still persists in the new server. 
I am assuming that when the conveyor breaks, that sponge's way of breaking the block, breaks the conveyor in such a way that it gets "renamed" and no longer functions as a conveyor. You get that "purple & black" texture. Since it became a new name, you cannot use them to build the press or any other structure with conveyors. 
Hope this helps.  I will confirm later with another test server and take out Ti-Quality to see if that fixes it
